adhocteam/Head-Start-TTADP
# TTAHUB-188: Changes for Sequelize Join Issue

## Description of change
Added the "separate: true" parameter to included models in the getActivityReportById function in the backend code

JIRA #: TTAHUB-188

## How to test
To start, we've created a complex activity report at https://tta-smarthub-dev.app.cloud.gov/activity-reports/10
This activity report is large enough to crash the dev instance simply by visiting the above URL. We've recorded some rough benchmarks for current performance.

Our testing strategy will be to recreate the report in the sandbox, perhaps in stages, and observe the behavior and responsiveness of the app in whatever monitoring tools are available in that environment (New Relic, Kibana, etc)

Once that's been done, we will move on to testing in dev and confirm performance is improved. Again, we will verify and record any findings via logging tools.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-188

## Checklists

### Every PR



**N/A**

### Production Deploy


### After merge/deploy
